### DISCUSSION TITLE
[Lerna monorepo] - upgrade different `depTypes` in the same PR.

### DISCUSSION THREAD

**Discussion by mdziadkowiec95** [renovatebot/renovate#19263] [Request Help]
Hi there!

I have such a problem with configuring Renovate to work properly in my Lerna based monorepo (javascript library)

**I would like to** have upgrades for each dependency type in the same PR. For example:

I have `@babel/core` dependency, which is declared as different type depending on the monorepo package.
It occurs as `dependency` , `devDependency` and also `peerDependency`. Beacuse of that, when upgrading `@babel/core` I want to make upgrade in each place (each dependency type). 

**Currently** renovate is  creating separate PR for each dependency type, so it this example I would end up with 3 different PRs for `@babel/core`, like below:

1. Upgrade @babel/core dependency to 8.0.1 
2. Upgrade @babel/core devDependency to 8.0.1 
3. Upgrade @babel/core peerDependency to 8.0.1 

Has anyone faces such problem to solve in monorepo project? Is so then I would appreciate some tips on how to approach it in such case.

Thanks in advance!

**Comment by rarkins** [comment]
They should be upgraded together by default, so maybe your config has split them. What are the branch names for those 3 PRs?
